### DISCUSSION TITLE
getEiStatus stato invio fattura elettronica

### DISCUSSION THREAD

**Discussion by zofe** [fattureincloud/api#183] [Q&A] [answered]
l'esigenza è quella di determinare, dalla chiamata all'elenco delle fatture, se ognuna è stata inviata elettronicamente o meno.

il **$invoice->getEiStatus**() via php sdk  non restituisce nessun dato, è questo il medoto/campo  corretto  da utilizzare?

grazie

**Comment by emajo** [comment, marked as answer]
Ciao @zofe, si, **$invoice->getEiStatus()** è il metodo corretto da utilizzare, se non restituisce nulla i casi sono 2:

- il documento non è elettronico
- non è stato impostato il [fieldset](https://developers.fattureincloud.it/docs/basics/additional-capabilities/customize-response/#-available-fieldsets) a **detailed**

Grazie,
Emanuele.
